float64 x_delta_ref
float64 y_delta_ref
float64 z_delta_ref
float64 action_time
float64[7] tau_null
bool tau_null_bool
---
bool success
string message
string error

